client/my-sites/site-settings/site-tools/index.jsx

```diff
@@ -104,13 +98,11 @@ class SiteTools extends Component {
 						description={ translate( "Register a new domain or change your site's address." ) }
 					/>
 				) }
-				{ isEnabled( 'sites/copy-site' ) && canSiteBeCopied && (
+				{ isEnabled( 'sites/copy-site' ) && shouldShowSiteCopyItem && (
 					<SiteToolsLink
 						href={ copySiteUrl }
 						onClick={ () => {
-							clearSignupDestinationCookie();
-							setPlanCartItem( { product_slug: planSlug } );
-							recordTracksEvent( 'calypso_settings_copy_site_option_click' );
+							startSiteCopy( 'calypso_settings_copy_site_option_click' );
```

Comment:
One nit is that this should maybe be an object of options, instead of passing the event name as the first argument.

We could potentially do this later, though.